14 Arrays and Loops/ArraysAndLoops.playground/Pages/Exercise-Screening Messages.xcplaygroundpage/Contents.swift

```diff
@@ -22,7 +22,12 @@ If the message refers to the Caterpillar, print it to the console.
  The `contains` method is part of the `Foundation` framework that you read about in the “Types” playground. If you try using it and get an error saying “Value of type 'String' has no member 'contains',” follow the instructions from that playground to import the framework into your project.
  */
 // Write the `for…in` loop here:
-
+for message in aliceMessages {
+    if message.contains("Caterpillar"){
```

Comment:
Another option:
```swift
    guard message.contains("Caterpillar") else { continue }
    print(message)
```

Reply:
what does guard mean?

Reply:
I wasn't sure if you'd gotten to it yet, but it's like an `if` that forces you to early return. If it doesn't make sense to you, just take a note of it and we can cover it at our next meeting.